tensorflow/python/BUILD

```diff
@@ -185,10 +188,11 @@ tf_py_test(
     tags = ["notap"],
 )
 
-cc_library(
+tf_cc_library(
     name = "cost_analyzer_lib",
     srcs = ["grappler/cost_analyzer.cc"],
     hdrs = ["grappler/cost_analyzer.h"],
+    copts = tf_copts(),
```

Comment:
Here and below: these tf_copts() aren't needed, right?

Reply:
Yes, I'll remove them. Thanks for pointing that.